src/main/java/org/spongepowered/common/data/SpongeSerializationRegistry.java

```diff
@@ -1271,13 +1283,13 @@ public static void setupSerialization(Game game) {
         dataRegistry.registerValueProcessor(Keys.ART, new ArtValueProcessor());
         dataRegistry.registerValueProcessor(Keys.TARGETS, new WitherTargetsValueProcessor());
         dataRegistry.registerValueProcessor(Keys.TARGETS, new TargetsValueProcessor());
-        dataRegistry.registerValueProcessor(Keys.FIREWORK_EFFECTS, new EntityFireworkEffectsValueProcessor());
```

Comment:
You probably resolved this conflict incorrectly. Re-add these and squash back into your commit.